src/signatures/winternitz.rs

```diff
@@ -222,6 +228,18 @@ mod test {
     // The secret key
     const MY_SECKEY: &str = "b138982ce17ac813d505b5b40b665d404e9528e7";
 
+    fn signatures_to_script(signatures: Vec<Vec<u8>>) -> Script {
+        script! {
+          for signature in signatures {
+            if signature.len() == 1 {
+              { signature[0] }
+            } else {
+              { signature }
+            }
+          }
+        }
+    }
```

Comment:
This function can be removed, since we don't use this function anymore.

Reply:
DONE